core/shared/src/main/scala-2.x/zio/VersionSpecific.scala

```diff
@@ -1,42 +1,19 @@
 package zio
 
-import izreflect.fundamentals.reflection.Tags._
+import izreflect.fundamentals.reflection.Tags.Tag
 import izreflect.fundamentals.reflection.macrortti.{ LightTypeTag, LightTypeTagRef }
 
 private[zio] trait VersionSpecific {
 
-  type TaggedType[A] = Tag[A]
-  type TaggedTypeF[F[_]] = TagK[F]
-  type TaggedTypeF2[F[_, _]] = TagKK[F]
-  type TaggedTypeF3[F[_, _, _]] = TagK3[F]
-  type TaggedTypeF4[F[_, _, _, _]] = HKTag[ { type Arg[A0, A1, A2, A3] = F[A0, A1, A2, A3] } ]
```

Comment:
@adamgfraser There should probably be some way for the user to summon these things when they work with highly polymorphic code.
I think we should just alias the tags in izumi-reflect here:

```scala
type TagK[F[_]] = Tags.TagK[F]
type TagKK[F[_, _]] = Tags.TagKK[F]
type TagK3[F[_, _, _]] = Tags.TagK3[F]
```

And maybe synchronize the naming too and just switch to `Tag` everywhere:

```scala
@deprecated("Use zio.Tag[A]")
type Tagged[A] = Tags.Tag[A]
type Tag[A] = Tags.Tag[A]
```

```scala
@deprecated("Use zio.Tag[A]")
type Tagged[A] = ClassTag[A]
type Tag[A] = ClassTag[A]
```

Reply:
@neko-kai Added back the type aliases. I left it as `Tagged` to avoid breaking one more thing and kept the `TaggedF` because I think it is slightly more accessible to users without a functional programming background. Just my personal opinion and happy to change if others have a different point of view but figured would get the code out there first.

Reply:
@adamgfraser I think users without background would not have an intuition regarding `F` vs `K` - neither actually tells anything to someone unfamiliar, and if they do have cats background they might find `K` more intuitive since `K` suffix in cats means 'higher-kinded' (`MonoidK`, `SemigroupK`, `FunctorK` etc.)